requirements/base.txt

```diff
@@ -7,3 +7,5 @@ bok-choy==0.5.2
 pep8==1.5.7
 pylint==1.5.5
 edx-lint==0.5.1
+edx-opaque-keys==0.2.1
+mock==1.0.1
```

Comment:
@raeeschachar This is being used. Some of the tests written require bokchoy pages using these requirements.